drivers/staging/vc04_services/bcm2835-camera/mmal-common.h

```diff
@@ -13,6 +13,8 @@
  * MMAL structures
  *
  */
+#ifndef __MMAL_COMMON_H
+#define __MMAL_COMMON_H
```

Comment:
Do you need the extra underscores here and not in mmal_msg.h?

Reply:
We've a slight mismash with mmal_parameters.h having underscores with most of the others not. I'll unify them.